### PR TITLE
[CP-1087] Create S3 bucket for storing latest MuditaOS releases for internal releases

### PR DESCRIPTION
**Pull request by dkarski**
Jira: [CP-1087]

**Description**

This PRs could be tested locally when:
- mudita-center-server: https://github.com/mudita/mudita-center-server/pull/22 will be merged 
- or run pr with the above mudita-center-server pr locally

[CP-1087]: https://appnroll.atlassian.net/browse/CP-1087?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: packages/app/src/api/mudita-center-server/client.ts [hunk 1]
```diff
@@ -13,7 +13,12 @@ import {
 import { MuditaCenterServerRoutes } from "App/api/mudita-center-server/mudita-center-server-routes"
```
Comment (igorbezsmertnyi) [inline packages/app/src/api/mudita-center-server/client.ts:13]: It's looking good, but let's also think about how we can cover this functionality within our new code convention

**Review by igorbezsmertnyi — APPROVED** [review]

**Review by igorbezsmertnyi — APPROVED** [review]

**Review by wercix — COMMENTED** [review]
I checked it locally but got error while checking for update
`Overview: check for updates fail. Data: v2 argument isn't semantic version: 1.2.0-daily.2022.04.12`

**Comment by dkarski** [comment]
> I checked it locally but got error while checking for update `Overview: check for updates fail. Data: v2 argument isn't semantic version: 1.2.0-daily.2022.04.12`

please check that you have correct `MUDITA_CENTER_SERVER_URL` variables set locally

**Comment by dkarski** [comment]
> I checked it locally but got error while checking for update `Overview: check for updates fail. Data: v2 argument isn't semantic version: 1.2.0-daily.2022.04.12`

oh right!

 `1.2.0-daily.2022.04.12` isn't a semantic version, so the contract between Center and OS isn't delivered.
 
 Conclusions.
 
 Mechanism works. Os need fix daily version naming. I'm going to create for them ticket 👌

**Review by wercix — APPROVED** [review]

**Pull request merged**